src/resources/elements/primeDesignSystem/pcountdown-closebutton/pcountdown-closebutton.html

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="pcountdown-closebutton  ${ paused ? 'isPaused' : ''}">
```

Comment:
what about using the <template> as the parent container element? This way we get rid of the `div` element

Reply:
Done, note I also added a `closed` callback, cuz is more flexible and useful for the popup notification